packages/xcoobee-cookie-kit-core/src/configs.js

```diff
@@ -71,7 +71,10 @@ const animations = {
   unknown: "default",
 };
 
-const cssHref = "xcoobee-cookie-kit.min.css";
+const cssHrefTheme1 = "xcoobee-cookie-kit-theme-1.min.css";
```

Comment:
Since position applies to all themes, I think we should name the theme based on its color/style. I think `XcooBee` is a good name for the original theme. It's identifier in the `themes` array below could be `xcoobee`. The new theme should have an appropriate name too. Based on what I saw in the mock up in the Trello card, `neutral`, `grey`, or `gray` might be appropriate names. You know better than I do about what it actually looks like. So, would you pick an appropriate name?

Once we have theme names, then we can rename the CSS files to use them instead of using `1` or `2`. We can update any related configs (e.g., cssHrefTheme1). `popup` would be changed to `xcoobee` and `overlay` would be changed to whatever the new theme name is.